examples/ros2_demo/turtle_listener.rs

```diff
@@ -73,7 +73,7 @@ impl TurtleListener {
 
       for event in events.iter() {
         if event.token() == TurtleListener::THREAD_CONTROL_TOKEN {
-          while let Ok(ctrl) = tc_receiver.try_recv() {
+          if let Ok(ctrl) = tc_receiver.try_recv() {
```

Comment:
this is logically equivalent to what was here previously, but i'm guessing this doesn't match the original intent?

the `while` loop never loops because it either a) is `Err`, so the loop breaks or b) is `Ok` and hits an explicit `return` statement